elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -63,3 +67,42 @@ button[aria-expanded="true"] {
     @include pfe-accordion-variables($state: expanded, $theme: #{$context});
   }
 }
+
+/// ===========================================================================
+/// DISCLOSURE STYLES
+/// ===========================================================================
+
+:host([pfe-disclosure="true"]) {
+  button {
+    --pfe-accordion--triggerPadding: #{pfe-var(container-spacer)} calc(#{pfe-var(container-spacer)} * 1.5) #{pfe-var(container-spacer)} calc((#{pfe-var(container-spacer)} * 3) + 2px) !important;
+
+    &[aria-expanded="true"] {
+      padding-left: calc(( #{pfe-var(container-spacer)} * 3 ) - 1px ) !important;
+
+      &::before {
+        @include pfe-chevron($state: open, $position: before);
+        left: calc( #{pfe-var(container-spacer)} * 1.3125 ); // 21px.
+      }
+    }
+    &[aria-expanded="false"]::before {
+      @include pfe-chevron($state: closed, $position: before);
+    }
+
+    // Adjust contents position on hover.
+    &:hover {
+      padding-left: calc(( #{pfe-var(container-spacer)} * 3 ) - 1px ) !important;
+      --pfe-accordion--BorderLeftWidth: 4px;
+
+      &::before {
+        left: calc( #{pfe-var(container-spacer)} * 1.3125 ); // 21px.
+      }
+    }
+  }
+}
+
+:host([pfe-disclosure="true"].animating) {
+  border-left-color: #{pfe-color(ui-base)};
+  border-left-width: #{pfe-var(surface--border-width--heavy)};
```

Comment:
should this reset the value of `--pfe-accordion--BorderLeftWidth`

Reply:
No, changed that to the correct property `--pfe-accordion--BorderLeftWidth`